src/brouwers/shop/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+
+from brouwers.shop.models import ProductReview, MAX_RATING, MIN_RATING
+from brouwers.utils.widgets import StarRatingSelect
+
+
+class ProductReviewForm(forms.ModelForm):
+    text = forms.CharField(widget=forms.Textarea())
```

Comment:
why are you explicitly defining this here? `TextArea` is the default widget for `TextField`, so this entire declaration isn't needed, it'll be pulled from the model. This declaration also kills the default label from the model.

Reply:
The model has RichTextField.